easybuild/easyconfigs/e/EPA-ng/EPA-ng-0.3.8-GCC-11.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'MakeCp'
+
+name = 'EPA-ng'
+version = '0.3.8'
+homepage = 'https://github.com/pierrebarbera/epa-ng'
+description = """
+EPA-ng is a complete rewrite of the Evolutionary Placement Algorithm (EPA), previously
+implemented in RAxML. It uses libpll and pll-modules to perform maximum likelihood-based
+phylogenetic placement of genetic sequences on a user-supplied reference tree and alignment.
+"""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/pierrebarbera/epa-ng/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d1db23919f49cfad202b18623e5eb30c3d6cedcc1705b944221daea8131cbb74']
+
+builddependencies = [
+    ('CMake', '3.24.3'),
+    ('make', '4.3'),
+]
+
+files_to_copy = ['%(installdir)s/bin']
```

```suggestion
files_to_copy = ['bin']
```

(otherwise it's trying to copy **from** `%(installdir)s/bin`